cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1289,6 +1289,20 @@ EOF
     sed -i -e "s@{{ *metrics_cpu_per_node *}}@${metrics_cpu_per_node}@g" "${controller_yaml}"
     update-prometheus-to-sd-parameters ${controller_yaml}
   fi
+  if [[ "${ENABLE_CLUSTER_MONITORING:-}" == "stackdriver" ]] ||
+     ([[ "${ENABLE_CLUSTER_LOGGING:-}" == "true" ]] &&
```

Comment:
Just to be sure, we've incorporated building the monitored resource directly into heapster for the new resource types, right?

Reply:
Right.